napari/_vispy/_tests/test_vispy_pyramid.py

```diff
@@ -1,4 +1,6 @@
 import numpy as np
+import os
```

Comment:
LOL this should be `import sys` then just sys. `os.sys` is an alias for `sys`, and I think it's incidental.

Reply:
done